Carthage.xcodeproj/project.pbxproj

```diff
@@ -902,7 +902,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D0D1212619E878CC005E4BAA /* Debug.xcconfig */;
 			buildSettings = {
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
It drives me insane that Xcode puts these in each target instead of the project level.